### PR TITLE
feat: deployment config for oidc server

### PR DESCRIPTION
**Pull request by shaarful**
## Description
1. Clone bahis-infra repo with git clone
```bash
git clone https://github.com/road86/bahis-infra
```
2. Update submodule with 
 ```bash
   cd bahis-infra
   git submodule update --init --recursive
   ```
3. Run the docker compose

```bash
cd bahis-infra/auth/
sudo docker-compose build
sudo docker-compose up -d
```

- [x] I have read the road86 Contribution Guide.
- [x] I have checked all commit message styles match the requested structure.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] My changes generate no new warnings.
- [ ] I have made corresponding changes to the documentation.
- [ ] New database changes have been committed.

### CODE REVIEW

File: auth/nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#this config says that whenever you call /api/v1 forward the request to flask:5000 using uswgi. Flask is the name of the service that we will create in our docker-compose file
```
Comment (ChasNelson1990) [inline auth/nginx/nginx.conf]: Erm, this comment doesn't seem to be about this file?

File: auth/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# How to deploy
+1. Clone bahis-oidc in the same level as bahis-infra
+```bash
+git clone https://github.com/road86/bahis-oidc
```
Comment (ChasNelson1990) [inline auth/README.md]: Why not just use submodules like dash?

**Review by ChasNelson1990 — COMMENTED** [review]
Otherwise looks good

File: auth/nginx/nginx.conf (hunk 1 above)
Comment (shaarful) [reply auth/nginx/nginx.conf]: I have copied this file from maybe dash or deployment

File: auth/README.md (hunk 2 above)
Comment (shaarful) [reply auth/README.md]: Then all the applications will be submodules of bahis infra.

File: auth/nginx/nginx.conf (hunk 1 above)
Comment (ChasNelson1990) [reply auth/nginx/nginx.conf]: Just remove this comment then

File: auth/README.md (hunk 2 above)
Comment (ChasNelson1990) [reply auth/README.md]: I think that's probably the way to go. Are you happy doing that?

Comment (shaarful) [reply auth/README.md]: sure. i am doing

Comment (ChasNelson1990) [inline auth/README.md]: This needs updating now we are using submodules

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]

File: auth/README.md (hunk 2 above)
Comment (shaarful) [reply auth/README.md]: Ohh. Forgot to edit readme. doing

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**